tasks/in-house-ctf/task.py

```diff
@@ -0,0 +1,78 @@
+from inspect_ai import Task, task
+from inspect_ai.scorer import includes
+from inspect_ai.solver import (
+    generate,
+    system_message,
+    use_tools,
+)
+from inspect_ai.tool import bash
+
+from dataset import get_task_sample, DATA_DIR
+from template import SYSTEM_MESSAGE
+
+# maximum chat messages before giving up
+MAX_MESSAGES = 30
+
+# maximum seconds to run bash/python cmds
+CMD_TIMEOUT = 180
+
+
+def in_house_ctf_task_template(task_name: str) -> Task:
+    """
+    In order to take advantage of the Eval Suite functions, we define this task template and
+    explicitly define each task below. As each task uses different victim docker environments,
+    following the intercode-ctf example would not be feasible.
+    """
+    challenge_sample = get_task_sample(task_name)
+    challenge_system_message = SYSTEM_MESSAGE.substitute(
+        objective_prompt=challenge_sample.input,
+        challenge_addresses=challenge_sample.metadata["challenge_address"],
+    )
+    return Task(
+        dataset=[challenge_sample],
+        plan=[
```

Comment:
One other thing I'd like to see is making it clear to users how they can pass in their own `plan`. There is some work I need to do on plan serialisation to make this maximally convenient. Will ping back here with a suggestion when that is ready.